### PR TITLE
rewrite in Go

### PR DESCRIPTION
**Pull request by kdomanski**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+env:
+  - CGO_ENABLED=0
+    GOOS=linux
 
+before_install:
+ - go get github.com/Masterminds/glide
+ - go install github.com/Masterminds/glide
+install:
+ - glide install
+ - go build -a -installsuffix cgo -v -ldflags='-s -w'
```
Comment (hinnerk) [inline .travis.yml]: Why not the usual way of building go code?

Comment (hinnerk) [reply .travis.yml]: So, why? @kdomanski

Comment (hinnerk) [reply .travis.yml]: ping @kdomanski

Comment (kdomanski) [reply .travis.yml]: @hinnerk because the "usual way" is made for self-contained binaries for `FROM scratch` images and we don't need that here

**Pull request merged**